torchci/lib/drciUtils.ts

```diff
@@ -553,3 +564,16 @@ export async function isSameAuthor(
   // * Cherry picking
   return isSameEmail || isSameCommitUsername || isSamePrUsername;
 }
+
+export function isRunnerDisconnectionFailure(job: RecentWorkflowsData): boolean {
```

Comment:
I dont see this function actually called anywhere?

It would probably also be a better idea to do something similar as isInfraFlakyJob and add it to the flakyJobs list with the relevant info about runner disconnection so you don't have to make a new section